scripts/exgdas_atmos_verfrad.sh

```diff
@@ -214,7 +214,7 @@ export CHGRP_CMD=${CHGRP_CMD:-"chgrp ${group_name:-rstprod}"}
 rlist="saphir"
 for rtype in $rlist; do
   if compgen -G "$TANKverf_rad/*${rtype}*" > /dev/null; then
-    ${CHGRP_CMD} $TANKverf_rad/*${rtype}*
+    ${CHGRP_CMD} "$TANKverf_rad/*${rtype}*"
```

Comment:
Globs won't work inside of quotations. Try
`"${TANKverf_rad}/"*${rtype}*`

If the linter still complains at that, use
`"${TANKverf_rad}/"*"${rtype}"*`

The `compgen` commands are different, because they want it as a single argument that it will expand itself (and won't work without them). Same for `find`. But regular shell commands like chmod want the argument expanded into multiple arguments first.

Reply:
The first solution does pass the linter.